doc/manual/src/projects.md

```diff
@@ -458,6 +458,42 @@ Hydra can send email notifications when the status of a build changes.
 This provides immediate feedback to maintainers or committers when a
 change causes build failures.
 
+The feature can be turned on by adding the following line to `hydra.conf`
+
+``` conf
+email_notification = 1
+```
+
+By default, Hydra only sends email notifications if a previously successful
+build starts to fail. In order to force Hydra to send an email for each build
+(including e.g. successful or cancelled ones), the environment variable
+`HYDRA_FORCE_SEND_MAIL` can be declared:
```

Comment:
I'm not suuuuper keen about these magic environment variables, and would rather wish they were actual configuration  options (consider: email_notifications = no|yes|always) but we've got the code we've got :).